### PR TITLE
Filters for fødeland and tolkebehov

### PR DESCRIPTION
**Pull request by dragutin-nav**

### CODE REVIEW

File: src/enhetsportefolje/enhet-kolonner.tsx [hunk 1]
```diff
@@ -108,6 +112,25 @@ function EnhetKolonner({className, bruker, enhetId, filtervalg, valgteKolonner,
...
+            <TekstKolonne
+                className="col col-xs-2"
+                tekst={tolkBehovSpraak(filtervalg, bruker, tolkbehovSpraakData)}
+                skalVises={valgteKolonner.includes(Kolonne.TOLKEBEHOV_SPRAAK)}
+            />
+            <TekstKolonne
+                className="col col-xs-2"
+                skalVises={valgteKolonner.includes(Kolonne.TOLKEBEHOV_SIST_OPPDATERT)}
+                tekst={
+                    bruker.tolkBehovSistOppdatert && bruker.tolkBehovSistOppdatert
```
Comment (stianStensli) [inline src/enhetsportefolje/enhet-kolonner.tsx:129]: Dobble sjekk på samme variabel 🐛

File: src/minoversikt/minoversikt-kolonner.tsx [hunk 2]
```diff
@@ -108,6 +112,25 @@ function MinoversiktDatokolonner({className, bruker, enhetId, filtervalg, valgte
...
+            <TekstKolonne
+                className="col col-xs-2"
+                tekst={tolkBehovSpraak(filtervalg, bruker, tolkbehovSpraakData)}
+                skalVises={valgteKolonner.includes(Kolonne.TOLKEBEHOV_SPRAAK)}
+            />
+            <TekstKolonne
+                className="col col-xs-2"
+                skalVises={valgteKolonner.includes(Kolonne.TOLKEBEHOV_SIST_OPPDATERT)}
+                tekst={
+                    bruker.tolkBehovSistOppdatert && bruker.tolkBehovSistOppdatert
```
Comment (stianStensli) [inline src/minoversikt/minoversikt-kolonner.tsx:129]: Dobble sjekk på samme variabel 🐛

**Review by stianStensli — CHANGES_REQUESTED** [review]

File: src/minoversikt/minoversikt-kolonner.tsx (hunk 2 above)
Comment (dragutin-nav) [reply src/minoversikt/minoversikt-kolonner.tsx]: Fixed

File: src/ducks/ui/listevisning-selectors.ts [hunk 3]
```diff
@@ -71,10 +71,25 @@ export function getMuligeKolonner(filtervalg: FiltervalgModell, oversiktType: Ov
...
+                filtervalg.tolkBehovSpraak.length > 0)
+        );
+    };
+
     return ([] as Kolonne[])
-        .concat(addHvis(Kolonne.FODELAND, filtervalg.landgruppe!.length > 0))
-        .concat(addHvis(Kolonne.STATSBORGERSKAP, filtervalg.landgruppe!.length > 0))
-        .concat(addHvis(Kolonne.STATSBORGERSKAP_GYLDIG_FRA, filtervalg.landgruppe!.length > 0))
+        .concat(addHvis(Kolonne.FODELAND, filtervalg.landgruppe!.length > 0 || filtervalg.foedeland.length > 0))
+        .concat(addHvis(Kolonne.STATSBORGERSKAP, filtervalg.landgruppe!.length > 0 || filtervalg.foedeland.length > 0))
```
[the remark this answers is not in the document]
Comment (dragutin-nav) [reply src/ducks/ui/listevisning-selectors.ts:86]: yep yep, fixing..

File: src/ducks/ui/listevisning-selectors.ts [hunk 4]
```diff
@@ -1,6 +1,6 @@
-import {AppState} from '../../reducer';
-import {Kolonne, ListevisningState, OversiktType} from './listevisning';
-import {AktiviteterValg, FiltreringAktiviteterValg} from '../filtrering';
+import { AppState } from "../../reducer";
```
Comment (Mathiamu) [inline src/ducks/ui/listevisning-selectors.ts:1]: It seems you are using some local linting rules and not the ones in .prettierrc.json, could you check?

File: package.json [hunk 5]
```diff
@@ -80,11 +80,15 @@
         "react-day-picker": "7.4.10",
         "react-device-detect": "2.2.2",
         "react-dom": "17.0.2",
+        "react-multi-select-component": "^4.3.3",
```
[the remark this answers is not in the document]
Comment (dragutin-nav) [reply package.json]: fixed

[the remark this answers is not in the document]
Comment (Mathiamu) [reply package.json]: Almost, there's still two different versions of "react-transition-group" 😬

[the remark this answers is not in the document]
Comment (dragutin-nav) [reply package.json]: that one is not 'mine' :D Kidding, I will fix that as well

File: src/ducks/ui/listevisning-selectors.ts (hunk 4 above)
Comment (dragutin-nav) [reply src/ducks/ui/listevisning-selectors.ts]: Fixed, I can't find configuration for prettier..had to call it from terminal on each commit :(

File: src/ducks/ui/listevisning-selectors.ts [hunk 6]
```diff
@@ -149,6 +164,9 @@ export function getMuligeKolonner(filtervalg: FiltervalgModell, oversiktType: Ov
             )
         )
         .concat(addHvis(Kolonne.VEILEDER, oversiktType === OversiktType.enhetensOversikt))
+        .concat(addHvis(Kolonne.TOLKEBEHOV, tolkBehovErValgt()))
+        .concat(addHvis(Kolonne.TOLKEBEHOV_SPRAAK, tolkBehovErValgt()))
         .concat(addHvis(Kolonne.NAVIDENT, oversiktType === OversiktType.enhetensOversikt))
```
Comment (stianStensli) [inline src/ducks/ui/listevisning-selectors.ts:169]: En siste ting, jeg tror ikke det er lurt å skille kolonnen VEILEDER fra NAVIDENT da de egentlig viser den samme dataen

**Review by stianStensli — APPROVED** [review]

File: src/ducks/ui/listevisning-selectors.ts (hunk 6 above)
Comment (dragutin-nav) [reply src/ducks/ui/listevisning-selectors.ts]: Fixed

**Pull request merged**